requirements.txt

```diff
@@ -164,7 +164,7 @@ packaging==25.0
     #   google-cloud-error-reporting
     #   google-cloud-kms
     #   gunicorn
-pillow==11.2.1
+pillow==11.1.0
```

Comment:
Looks like this change should be reverted-- it's downgrading a package that seems unrelated to the issue